packages/cms-data-sync/test/news/news.data-migration.test.ts

```diff
@@ -0,0 +1,232 @@
+      id: 'news-1',
+      flatData: {
+        title: 'news',
+        text: '<p>Hello world</p>',
+      },
+    },
+  ],
+};
+
+describe('migrateNews', () => {
+  let contenfulEnv: Environment;
+  let squidexGraphqlClientMock: jest.Mocked<SquidexGraphqlClient>;
+
+  beforeEach(() => {
+    contenfulEnv = getContentfulEnvironmentMock();
+    squidexGraphqlClientMock = {
+      request: jest.fn(),
+    };
+
+    (getSquidexAndContentfulClients as jest.Mock).mockResolvedValueOnce({
+      contentfulEnvironment: contenfulEnv,
+      squidexGraphqlClient: squidexGraphqlClientMock,
+    });
+
+    jest.spyOn(contenfulEnv, 'getEntries').mockResolvedValueOnce({
+      total: 1,
+      items: [newsEntry],
+      skip: 0,
+      limit: 10,
+      toPlainObject: jest.fn(),
+      sys: { type: 'Array' },
+    });
+
+    jest
+      .spyOn(newsEntry, 'publish')
+      .mockImplementationOnce(() => Promise.resolve(newsEntry));
+  });
+
+  afterEach(() => {
+    jest.resetModules();
+    jest.clearAllMocks();
+  });
+
+  it('clears contentful entries', async () => {
+    squidexGraphqlClientMock.request.mockResolvedValueOnce(
+      squidexResponseWithTitleOnly,
+    );
+
+    await migrateNews();
+    const clearContentfulEntriesMock = clearContentfulEntries as jest.Mock;
+
+    expect(clearContentfulEntriesMock).toHaveBeenCalled();
+  });
+
+  describe('creates contentful entries', () => {
+    it('for a news that contains only title', async () => {
+      squidexGraphqlClientMock.request.mockResolvedValueOnce(
+        squidexResponseWithTitleOnly,
+      );
+
+      await migrateNews();
+
+      expect(contenfulEnv.createEntryWithId).toHaveBeenCalledWith(
+        'news',
+        'news-1',
+        {
+          fields: {
+            frequency: { 'en-US': 'News Articles' },
+            link: { 'en-US': undefined },
+            linkText: { 'en-US': undefined },
+            shortText: { 'en-US': undefined },
+            text: { 'en-US': null },
+            thumbnail: { 'en-US': null },
+            title: { 'en-US': 'news' },
+          },
+        },
+      );
+    });
+
+    it('for a news that contains thumbnail', async () => {
+      squidexGraphqlClientMock.request.mockResolvedValueOnce(
+        squidexResponseWithThumb,
+      );
+
+      await migrateNews();
+
+      const createAssetLinkMock = createAssetLink as jest.Mock;
+
+      expect(createAssetLinkMock).toHaveBeenCalled();
+    });
+
+    it('for a news that contains text', async () => {
+      squidexGraphqlClientMock.request.mockResolvedValueOnce(
+        squidexResponseWithText,
+      );
+      const convertHtmlToContentfulFormatMock =
+        convertHtmlToContentfulFormat as jest.Mock;
+
+      await migrateNews();
+
+      expect(convertHtmlToContentfulFormatMock).toHaveBeenCalled();
+    });
+  });
+
+  describe('error handling', () => {
+    it('outputs a message when converting from html to contenful format gives an error', async () => {
+      console.log = jest.fn();
```

Comment:
maybe do this in a beforeAll/Each and then make sure it's assigned back in an afterAll?

Reply:
committed here https://github.com/yldio/asap-hub/pull/2290/commits/4604b7aefa73f806801f038969bc5782e7d67a22